test/winter_onboarding_2021/fleet_management_service/db/fleet_test.clj

```diff
@@ -80,4 +100,4 @@
                                       :fleet-id (:fleets/id (first fleets))})
       (core-db/insert! :users-fleets {:user-id (:users/id manager2)
                                       :fleet-id (:fleets/id (first fleets))})
-      (is (= [manager1 manager2] (fleet-db/managers (first fleets)))))))
+      (is (= [(select-keys-user manager1) (select-keys-user manager2)] (fleet-db/managers (first fleets)))))))
```

Comment:
we can add `invite-id` while selecting managers in fleet-db instead of using select-keys-user for both managers.

Reply:
Changed the format of select-keys instead. Using mapv on managers